plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/OpenShiftServerExtendedProperties.java

```diff
@@ -150,4 +154,25 @@ protected IStatus run(IProgressMonitor monitor) {
 		}.schedule();
 	}
 
+	
```

Comment:
Again, I don't think this should really belong here. My concern is that extended properties simply becomes a dumping ground for anything we want to expose. It's better to simply use the method where it currently exists.